models/Users.js

```diff
@@ -0,0 +1,42 @@
+const bcrypt = require('bcrypt');
+const { DataTypes } = require('sequelize');
+const sequelize = require('../config/database');
```

Comment:
everytime you will call the database file. It will create a new sequelize instance and return it. So, this means you are creating multiple database connections. Which will result in database overhead